utils/validations/test_validate_sarif.go

```diff
@@ -0,0 +1,242 @@
+package validations
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/jfrog/jfrog-cli-security/utils/formats/sarifutils"
+	"github.com/jfrog/jfrog-cli-security/utils/jasutils"
+	"github.com/jfrog/jfrog-cli-security/utils/results/conversion/sarifparser"
+	"github.com/owenrumney/go-sarif/v2/sarif"
+	"github.com/stretchr/testify/assert"
+)
+
+// Content should be a Json string of sarif.Report and will be unmarshal.
+// Value is set as the Actual content in the validation params
+func VerifySarifResults(t *testing.T, content string, params ValidationParams) {
```

Comment:
not in use

Reply:
if you want to validate content in sarif format it can be used, we just don't have integration tests for that like we have for Json and SimpleJson formats...